source/quickstart.txt

```diff
@@ -219,6 +220,34 @@ The API documentation provides details on using the following endpoints:
    * - :ref:`c2c-api-reverse`
      - Reverses the direction of a committed sync operation.
 
+One-Time Sync
+-------------
+
+For a one-time sync, after initializing data synchronization, use the
+:ref:`progress <c2c-api-progress>` command to see the status of the
+synchronization process: 
+
+.. literalinclude:: /includes/api/requests/progress.sh
+   :language: shell
+
+The following requirements must be met for a one-time sync: 
```

Comment:
[suggestion] active voice - I think we can reword this to be more active 

Maybe something along the lines of: 
"For a one-time sync, verify that the ``progress`` response includes the following field values: "